docs/get-started/installing-cardano-wallet.md

```diff
@@ -14,11 +14,11 @@ In this guide, we will show you how to compile and install `cardano-wallet` into
 
 :::note
 
-If you want to avoid compiling the binaries yourself, You can download the latest pre-built binaries of `cardano-wallet` from the links below. 
+If you want to avoid compiling the binaries yourself, You can download the latest pre-built binaries from the [`cardano-wallet`](https://github.com/cardano-foundation/cardano-wallet) github repository.
```

Comment:
```suggestion
If you want to avoid compiling the binaries yourself, you can download the latest pre-built binaries from the GitHub repository:  [`cardano-wallet` Releases](https://github.com/cardano-foundation/cardano-wallet/releases)
```